portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -12,7 +14,29 @@ public class HelloWorldServlet extends HttpServlet {
   //GET request handler
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Cindy!</h1>");
+    ArrayList<String> comments = new ArrayList<String>();
+    comments.add("I'm learning a lot of new concepts!");
+    comments.add("Everybody is very supportive!");
+    comments.add("Relearning Java right now");
```

Comment:
Consider:

```
List<String> strings = List.of("foo", "bar", "baz");
```

Also, should this be in your doGet? This means that this list will be initialized every time this method gets called. Initializing a list is an expensive operation. Maybe move it outside the method?

Also, if this list will never change, consider making it immutable: https://www.geeksforgeeks.org/immutable-list-in-java/#:~:text=New%20ImmutableList%20using%20of()%20function%20from%20Guava